src/Symfony/Bridge/PhpUnit/bin/simple-phpunit

```diff
@@ -126,9 +126,14 @@ if (!class_exists('SymfonyBlacklistPhpunit', false)) {
 if (class_exists('PHPUnit_Util_Blacklist')) {
     PHPUnit_Util_Blacklist::$blacklistedClassNames['SymfonyBlacklistPhpunit'] = 1;
     PHPUnit_Util_Blacklist::$blacklistedClassNames['SymfonyBlacklistSimplePhpunit'] = 1;
-} else {
-    PHPUnit\Util\Blacklist::$blacklistedClassNames['SymfonyBlacklistPhpunit'] = 1;
-    PHPUnit\Util\Blacklist::$blacklistedClassNames['SymfonyBlacklistSimplePhpunit'] = 1;
+} elseif (class_exists(\PHPUnit\Util\Blacklist::class)) {
```

Comment:
we need this check, better remove it and remove one indentation level (same in the other file - no need for the $className variable then)

Reply:
@nicolas-grekas 

Why not? The class is marked as `@internal` and can be removed or renamed at any time.

Reply:
It's not internal since [9.1.3](https://github.com/sebastianbergmann/phpunit/blob/b3c4b198184ac76710efe1258598c559dff6e401/ChangeLog-9.1.md#913---2020-04-23)

Reply:
So, what should I do? Keep it as is or change it?

Reply:
change it please